blocks/cards/cards.css

```diff
@@ -421,3 +430,56 @@
   background-color: var(--bg-text-light-plum);
   color: var(--c-dark-plum);
 }
+
+:root {
+    --c-midnight-blue: #000048;
```

Comment:
as far as I can see we have this already defined: https://github.com/Netcentric/cn-website/blob/main/styles/styles.css#L43